packages/clay-css/src/scss/components/_modals.scss

```diff
@@ -1,3 +1,7 @@
+.modal {
+	pointer-events: none;
```

Comment:
hey @pat270 let me know if I'm adding in the right place or if we will have implications with that.

Reply:
After conversation with @pat270 this would not be recommended as it can not work with bootstrap modal plugin. So I followed another, simpler approach that solves the problem 🙂.